impeller/entity/contents/solid_rrect_blur_contents.cc

```diff
@@ -48,13 +49,20 @@ Color SolidRRectBlurContents::GetColor() const {
   return color_;
 }
 
+namespace {
+Scalar ScaleSigma(Scalar sigma) {
+  // 0.5 comes from the math in `blur_rrect.frag`.
+  return 0.5 * GaussianBlurFilterContents::ScaleSigma(sigma);
```

Comment:
@bdero The majority of this fix comes from 0.5.  I double checked this isn't a case of misusing the content scalar.  This looks good at 1x (golden tests) and 2x (playgrounds).

I double checked `rrect_blur.frag`.  I don't have a good intuition why the math there is wrong.  We could move this scalar into the fragment shader if we wanted.  One guess is that we aren't accounting for the fact that we are sampling 4 times which is throwing off the gaussian scalar.  I'm happy to walk through it with you to brainstorm what's going on there.

Reply:
outdated